src/tests/multihost/alltests/test_sssctl_analyzer.py

```diff
@@ -30,6 +30,12 @@ def analyze(multihost, req_arg, op_arg=None):
     return ss_cmd.returncode, ss_cmd.stdout_text
 
 
+def execute_cmd(multihost, command):
```

Comment:
This function is very limited and does not allow to disable raising exception and adding stdin input.
It might be better to use the  multihost.client[0].run_command directly.
It would also improve readability of the code.

Reply:
ok removed